xdsl/ir.py

```diff
@@ -274,7 +275,26 @@ def irdl_definition(cls) -> ParamAttrDef:
 
 
 @dataclass
-class Operation:
+class Node(object):
```

Comment:
I am still against the name `Node`. I prefer `IRConstruct` or something in this direction.